src/components/SavedNewsHeader/SavedNewsHeader.js

```diff
@@ -1,10 +1,41 @@
 import './SavedNewsHeader.css'
 import { Navigation } from '../Navigation/Navigation'
-import { useContext } from 'react'
+import { useContext, useEffect, useState } from 'react'
 import { userDataContext } from '../../contexts/UserInfoContext'
```

Comment:
`userDataContext` should be in upper case `UserDataContext` because it acts as a react component in the markup